data-prepper-core/src/main/java/com/amazon/dataprepper/DataPrepper.java

```diff
@@ -69,20 +74,44 @@ private DataPrepper() {
         if (dataPrepper != null) {
             throw new RuntimeException("Please use getInstance() for an instance of this Data Prepper");
         }
-        startPrometheusBackend();
+        startMeterRegistryForDataPrepper();
         dataPrepperServer = new DataPrepperServer(this);
     }
 
-    public static PrometheusMeterRegistry getSysJVMMeterRegistry() {
-        return sysJVMMeterRegistry;
-    }
-
     /**
-     * Create a PrometheusMeterRegistry for this DataPrepper and register it with the global registry
+     * Creates instances of configured MeterRegistry and registers to {@link Metrics} globalRegistry to be used by
+     * Meters.
      */
-    private static void startPrometheusBackend() {
-        final PrometheusMeterRegistry prometheusMeterRegistry = new PrometheusMeterRegistry(PrometheusConfig.DEFAULT);
-        Metrics.addRegistry(prometheusMeterRegistry);
+    private static void startMeterRegistryForDataPrepper() {
+        final List<MetricRegistryType> configuredMetricRegistryTypes = configuration.getMetricRegistryTypes();
+        configuredMetricRegistryTypes.forEach(DataPrepper::createAndAttachMeterRegistry);
+    }
+
+    private static void createAndAttachMeterRegistry(final MetricRegistryType metricRegistryType) {
+        switch (metricRegistryType) {
+            case Prometheus:
+                Metrics.addRegistry(new PrometheusMeterRegistry(PrometheusConfig.DEFAULT));
+                break;
+            case CloudWatch:
+                Metrics.addRegistry(new CloudWatchMeterRegistryProvider().getCloudWatchMeterRegistry());
+                break;
+            default:
+                throw new IllegalArgumentException(format("Invalid metricRegistryType %s", metricRegistryType));
+        }
```

Comment:
Just now noticed this - could we remove this switch block in favor of `MetricRegistryType::getDefaultMeterRegistryForType`?